web/main/templates/includes/breadcrumbs.html

```diff
@@ -11,5 +12,15 @@
         <span class="separator">.</span>
       {% endif %}
     {% endfor %}
+    {% if content.reading_time %}
+      <div class="reading-time {% if editing %}reading-time-tooltip{% endif %}" title="Read time estimate is visible to readers">
+        {{ content.reading_time | humanize_minutes }}
+        {% if content.num_links %}
+        + {{ content.num_links }} link{% if content.num_links > 1 %}s{% endif %}
```

Comment:
Django has a built-in for this!

https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#pluralize

(Though honestly it's barely fewer characters.)